packages/stack/src/types.tsx

```diff
@@ -396,15 +396,15 @@ export type StackCardInterpolationProps = {
   /**
    * Animated node representing whether the card is closing (1 - closing, 0 - not closing).
    */
-  closing: Animated.AnimatedInterpolation;
+  closing: Animated.AnimatedInterpolation<0 | 1>;
   /**
    * Animated node representing whether the card is being swiped (1 - swiping, 0 - not swiping).
    */
-  swiping: Animated.AnimatedInterpolation;
+  swiping: Animated.AnimatedInterpolation<0 | 1>;
   /**
    * Animated node representing multiplier when direction is inverted (-1 - inverted, 1 - normal).
    */
-  inverted: Animated.AnimatedInterpolation;
+  inverted: Animated.AnimatedInterpolation<1 | -1>;
```

Comment:
```suggestion
  inverted: Animated.AnimatedInterpolation<-1 | 1>;
```

Reply:
Just to be consistent with other types

Reply:
Thanks! 🙌 
Fixed in [7d7b3f7](https://github.com/react-navigation/react-navigation/pull/11421/commits/7d7b3f7a23dc758866b3c501cb1868b7d09ee8ee)